libnavui-androidauto/src/main/java/com/mapbox/androidauto/navigation/speedlimit/SpeedLimitDrawable.kt

```diff
@@ -0,0 +1,279 @@
+package com.mapbox.androidauto.navigation.speedlimit
+
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.ColorFilter
+import android.graphics.Paint
+import android.graphics.PixelFormat
+import android.graphics.Rect
+import android.graphics.RectF
+import android.graphics.drawable.Drawable
+import androidx.annotation.ColorInt
+
+internal abstract class SpeedLimitDrawable : Drawable() {
```

Comment:
nitpick:

Will you move each of these classes to a separate file? There is a designated package and ideally we could have tests for each of these classes

Reply:
Sure.
We have instrumentation tests that verify resulting bitmaps so there is no need to unit-test these `Drawables`. Besides, how would you unit-test them? Verify `Canvas` calls?

Reply:
Yeah it's reason there are bitmap image compare tests, instead of unit tests for canvas calls.   Resolving here